src/tasks/__tests__/processBulkUpload.int.test.ts

```diff
@@ -402,6 +438,18 @@ describe('processBulkUpload', () => {
 			createdAt: expect.any(Date) as Date,
 		});
 
+		const [proposalSubmitterEmailFormField, organizationNameFormField] =
+			await getApplicationFormFieldsByBaseFieldIds(1, [
```

Comment:
(Essentially transcribing our pair review…) Here you're referencing the `ApplicationForm`'s ID statically. Can you (1) test that the `ApplicationForm` was actually created, and (2) reference its ID instead of statically including it? Like you just did with your `createTestBaseFields()` function.

Reply:
Fixed up!